lib/rover/data_frame.rb

```diff
@@ -436,9 +436,10 @@ def plot(x = nil, y = nil, type: nil)
           .mark(type: type, tooltip: true, interpolate: "cardinal", point: {size: 60})
           .encoding(
             x: {field: x, type: x_type, scale: scale},
-            y: {field: y, type: "quantitative"}
+            y: {field: y, type: "quantitative"},
+            color: {field: group}
           )
-          .config(axis: {labelFontSize: 12})
+          .config(axis: {title: nil, labelFontSize: 12})
```

Comment:
I think we'll want to keep the titles.

Reply:
Should I make it such that we can use variable parameters to decide whether to add title or not?

Reply:
Users should be able to call `config(axis: {title: nil})` on the returned Vega object, so I don't think an option is needed right now.